### PR TITLE
[11.0][ADD] product_secondary_unit: Move sale_secondary_unit to sale_order_secondary_unit module

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

File: product_secondary_unit/views/product_views.xml [hunk 1]
```diff
@@ -14,6 +14,11 @@
                        domain="[('product_tmpl_id', '=', id)]"
                        options="{'no_create': True}"/>
             </xpath>
+            <xpath expr="//field[@name='uom_po_id']" position="after">
```
Comment (nikul-serpentcs) [inline product_secondary_unit/views/product_views.xml]: IMO here Direct use `<field name="uom_po_id" position="after">`

**Comment by sergio-teruel** [comment]
@nikul-serpentcs Changes done!! Thanks

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

File: product_secondary_unit/models/product_template.py [hunk 2]
```diff
@@ -16,3 +16,7 @@ class ProductTemplate(models.Model):
         comodel_name='product.secondary.unit',
         string='Default unit sale',
     )
+    purchase_secondary_uom_id = fields.Many2one(
```
Comment (lmignon) [inline product_secondary_unit/models/product_template.py]: @sergio-teruel Why do we have a purchase field into an addon that does not depend on 'purchase'?  This field should be declared into `purchase_order_secondary_unit`

**Review by lmignon — CHANGES_REQUESTED** [review]

File: product_secondary_unit/models/product_template.py (hunk 2 above)
Comment (lmignon) [reply product_secondary_unit/models/product_template.py]: The same comment applies to `sale_secondary_uom_id` This one should be declared into `sale_order_secondary_unit`. Do I miss something?

Comment (sergio-teruel) [reply product_secondary_unit/models/product_template.py]: I desagree with that. As yo see here (https://github.com/OCA/OCB/blob/11.0/addons/product/models/product_template.py#L104) is in produt module where is declared this field, so I have done the same..

Comment (lmignon) [reply product_secondary_unit/models/product_template.py]: I'll not block but I also find not logic to have these 'sale' ant 'purchase' fields of odoo core declared into product. If we can avoid the same mistake...

**Comment by sergio-teruel** [comment]
@lmignon Changes done.. I change each field in your module...

**Review by lmignon — APPROVED** [review]
Thank you @sergio-teruel !

**Comment by rousseldenis** [comment]
@sergio-teruel Is the title of this PR relevant towards changes ?

**Comment by sergio-teruel** [comment]
Noo... ups!! Ok I removed the sale secondary unit. I moved it to sale_order_secondary_unit module...

**Comment by sergio-teruel** [comment]
@rousseldenis Title changed!! Thanks

File: product_secondary_unit/i18n/es.po [hunk 3]
```diff
@@ -39,6 +39,14 @@ msgstr "Creado en"
 msgid "Default Secondary Unit of Measure."
 msgstr "Segunda unidad de medida por defecto"
 
+#. module: product_secondary_unit
```
Comment (rousseldenis) [inline product_secondary_unit/i18n/es.po:42]: @sergio-teruel I think this is not necessary anymore

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**